code/src/components/AddTodoStyled.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
```

Comment:
I like that you're using separat styled components, it makes it easier to see the "functional" code without being distracted by styling.